### PR TITLE
Update GMSA docs for beta in v1.16

### PR DESCRIPTION
**Pull request by ddebroy**
Update to docs to reflect Beta state of GMSA feature in v1.16

This is a PR against master and supersedes https://github.com/kubernetes/website/pull/16464

### CODE REVIEW

**Comment by ddebroy** [comment]
@tengqm @steveperry-53 FYI

File: content/en/docs/tasks/configure-pod-container/configure-gmsa.md [hunk 1]
```diff
@@ -6,24 +6,20 @@ weight: 20
...
 
-### Enable the WindowsGMSA feature gate
-In the alpha state, the `WindowsGMSA` feature gate needs to be enabled on kubelet on Windows nodes. This is required to pass down the GMSA credential specs from the cluster scoped configurations to the container runtime. See [Feature Gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) for an explanation of enabling feature gates. Please make sure `--feature-gates=WindowsGMSA=true` parameter exists in the kubelet.exe command line.
+### WindowsGMSA feature gate
+In the beta state, the `WindowsGMSA` feature gate (required to pass down GMSA credential specs from the pod specs to the container runtime) is enabled by default on the API server and Kubelet. See [Feature Gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) for an explanation of enabling feature gates in prior releases.
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/configure-gmsa.md]: nit and tweak:
```suggestion
The `WindowsGMSA` feature gate (required to pass down GMSA credential specs from the pod specs to the container runtime) is enabled by default on the API server and the kubelet. See [Feature Gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) for an explanation of enabling or disabling feature gates.
```

Comment (ddebroy) [reply content/en/docs/tasks/configure-pod-container/configure-gmsa.md]: Thanks - done!

**Comment by ddebroy** [comment]
/hold
Need a couple of more updates from @jeremywx around a troubleshooting step when networking is not up as soon as the entrypoint is initialized.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-16579--kubernetes-io-master-staging.netlify.com/docs/tasks/configure-pod-container/configure-gmsa/

File: content/en/docs/tasks/configure-pod-container/configure-gmsa.md [hunk 2]
```diff
@@ -199,4 +199,60 @@ As pod specs with GMSA fields populated (as described above) are applied in a cl
 
 1. The container runtime configures each Windows container with the specified GMSA credential spec so that the container can assume the identity of the GMSA in Active Directory and access services in the domain using that identity.
 
+## Troubleshooting
+
+If you are having difficulties getting gmsa to work in your environment, they are a few troubleshooting steps you can take.
+
```
Comment (kbhawkey) [inline content/en/docs/tasks/configure-pod-container/configure-gmsa.md]: hello @ddebroy . Several nits in this section of the page:
- capitalize GMSA
- capitalize Pod
- remove `we`, lines 206 and 219
In the example below, the Pod ...
If your pod ... then next check communication with the ...

File: content/en/docs/tasks/configure-pod-container/configure-gmsa.md [hunk 3]
```diff
@@ -6,24 +6,20 @@ weight: 20
...
 
-In Kubernetes, GMSA credential specs are configured at a Kubernetes cluster-wide scope as custom resources. Windows pods, as well as individual containers within a pod, can be configured to use a GMSA for domain based functions (e.g. Kerberos authentication) when interacting with other Windows services. As of v1.14, the only container runtime interface that supports GMSA for Windows workloads is Dockershim. Implementation of GMSA through CRI and other runtimes is planned for the future.
-
-{{< note >}}
-Currently this feature is in alpha state. While the overall goals and functionality will not change, the way in which the GMSA credspec references are specified in pod specs may change from annotations to API fields. Please take this into consideration when testing or adopting this feature.
-{{< /note >}}
+In Kubernetes, GMSA credential specs are configured at a Kubernetes cluster-wide scope as custom resources. Windows pods, as well as individual containers within a pod, can be configured to use a GMSA for domain based functions (e.g. Kerberos authentication) when interacting with other Windows services. As of v1.16, the Docker runtime supports GMSA for Windows workloads.
```
Comment (kbhawkey) [inline content/en/docs/tasks/configure-pod-container/configure-gmsa.md]: nits:
line 11, capitalize Pods?
line 13, capitalize Custom Resources, Pods, Pod?

**Comment by ddebroy** [comment]
@kbhawkey I have addressed your comments

File: content/en/docs/tasks/configure-pod-container/configure-gmsa.md [hunk 4]
```diff
@@ -6,24 +6,20 @@ weight: 20
...
 
-In Kubernetes, GMSA credential specs are configured at a Kubernetes cluster-wide scope as custom resources. Windows pods, as well as individual containers within a pod, can be configured to use a GMSA for domain based functions (e.g. Kerberos authentication) when interacting with other Windows services. As of v1.14, the only container runtime interface that supports GMSA for Windows workloads is Dockershim. Implementation of GMSA through CRI and other runtimes is planned for the future.
-
-{{< note >}}
-Currently this feature is in alpha state. While the overall goals and functionality will not change, the way in which the GMSA credspec references are specified in pod specs may change from annotations to API fields. Please take this into consideration when testing or adopting this feature.
-{{< /note >}}
+In Kubernetes, GMSA credential specs are configured at a Kubernetes cluster-wide scope as Custom Resources. Windows Pods, as well as individual containers within a Pod, can be configured to use a GMSA for domain based functions (e.g. Kerberos authentication) when interacting with other Windows services. As of v1.16, the Docker runtime supports GMSA for Windows workloads.
```
Comment (kbhawkey) [inline content/en/docs/tasks/configure-pod-container/configure-gmsa.md]: @ddebroy , I read through the page again. A few more nits:
- line 13: To clarify, the last sentence is: As of Kubernetes v1.16, the Docker runtime ...? Or are you referring to the Docker version here?
- line 22: change the docs link to the feature gates page (minus the backticks):
`See [Feature Gates](/docs/reference/command-line-tools-reference/feature-gates/) for an explanation of enabling or disabling feature gates.`
- line 25: change the docs link to the CRD page:
`A [CustomResourceDefinition](/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/)`
- Try to capitalize the word Pod throughout the page. There are just a few places remaining.
- For consistency, change the instances of "use" verb to `use` verb?
- line 204: Should be changed to: If you are having difficulties getting GMSA to work in your environment, there are a few troubleshooting steps you can take:
  OR: If you are having difficulties getting GMSA to work in your environment, you can follow these steps to troubleshoot your setup:

Comment (ddebroy) [reply content/en/docs/tasks/configure-pod-container/configure-gmsa.md]: Line 13, the last sentence is indeed `As of Kubernetes v1.16, the Docker runtime`. Not referring to Docker versions.

Addressed the other nits. Thanks for reviewing!

**Pull request merged**